tensorflow_probability/python/mcmc/dual_averaging_step_size_adaptation.py

```diff
@@ -0,0 +1,507 @@
+# Copyright 2019 The TensorFlow Probability Authors.
```

Comment:
One more small issue, could you change this to be 2018? Our import process breaks otherwise... sorry for missing this earlier!

Reply:
Sorry for the chatter but we should definitely not put the wrong year here. We will come up with a fix internally.

Reply:
This should be good to go now (with 2019 as the date)